apps/condo/domains/meter/schema/RegisterMetersReadingsService.js

```diff
@@ -369,10 +380,10 @@ const RegisterMetersReadingsService = new GQLCustomSchema('RegisterMetersReading
                                 controlReadingsDate: rawControlReadingsDate ? toISO(rawControlReadingsDate) : dayjs().toISOString(),
                             })
                             meterId = createdMeter.id
-                        } catch (e) {
-                            resultRows.push(e)
-                            continue
                         }
+                    } catch (e) {
+                        resultRows.push(e)
```

Comment:
Why you just ignore this error? No logs?

Reply:
This error is not ignored. The client should process it. I think we do not need to log thousands lines from huge registries.